### DISCUSSION TITLE
(test) 점심메뉴는 짜장인가 짬뽕인가?

### DISCUSSION THREAD

**Discussion by sihyung92** [woowacourse-tobi-spring-study/tobi-vol1#7] [Show and tell]
## 🍜 짜장 짬뽕 중 무엇을 먹어야 할까?
  - 당신의 선택은?
  - 요즘처럼 비가 추적추적 내리는 날 따끈한 짬뽕 한 그릇이 주는 감동
  - 스테디셀러, 후루룩후루룩 정신 못 차리고 먹는 달콤한 짜장면

> 정답은 둘 다 먹는다 이다

**Comment by da-nyee** [comment]
나는 간짜장 🙃

**Reply by binghe819** [reply]
전 차돌 짬뽕 :)
